src/components/quiz/UserDetailsForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+interface User {
+  name: string | null;
+  displayPic: string | null;
+}
+
+export const UserDetailsForm: React.FunctionComponent = ({status,setStatus}: any) => {
+
+  const [userDetails, setUserDetails] = useState<User>({
+    name: null,
+    displayPic:null,
+  });
+  
+
+  function onSubmit(event: any) {    
+    event.preventDefault();     
+    setStatus(true);   
+  }
+
+  return (
+    <div>
+      <h2>Enter your details cadet !</h2>
```

Comment:
No space before punctuation
```suggestion
      <h2>Enter your details cadet!</h2>
```